sidekick_core/lib/src/commands/update_command.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:sidekick_core/sidekick_core.dart';
+import 'package:sidekick_core/src/sidekick_version_checker.dart';
+
+/// Updates the sidekick cli
+class UpdateCommand extends Command {
+  @override
+  final String description = 'Updates the sidekick cli';
+
+  @override
+  final String name = 'update';
+
+  @override
+  Future<void> run() async {
+    const sidekickVersionChecker = SidekickVersionChecker();
+    final latestSidekickVersion =
+        await sidekickVersionChecker.getLatestPackageVersion('sidekick');
```

Comment:
We might ship updates to package `sidekick` that do not change the generated code. Maybe we should create a `template_version`